Add quick start section to readme #346 (#348)
WordPress/performance: README.md

[Learn more about the Performance Lab plugin.](https://make.wordpress.org/core/2022/03/07/the-performance-lab-plugin-has-been-released/)

## Quick Start

> In order to get started contributing to the project you have to follow the next steps:

Reviewer: Reads a little awkward; how about "To get started with contributing to the Performance Lab project:"

Author: 👍

1. Fork the repository.
2. Clone the fork locally.
3. Run `composer install` in the project folder.
4. Run `npm install` in the project folder.

> 5. Start the development environment by running `npm run wp-env start`.
> 6. Login using `admin` and `password`.

Reviewer 2: I think we should also say something about where the user needs to go in order to log in. Maybe like this?
```suggestion
5. Start the development environment by running `npm run wp-env start`.
6. Go to `http://localhost:8888/wp-admin/` in your browser.
7. Log in using `admin` and `password`.
```

> The details about how the environment works can be found here [here](https://make.wordpress.org/core/2020/03/03/wp-env-simple-local-environments-for-wordpress/) <br>

Reviewer: Add period to end of sentence

> The details about how the environment works can be found here [here](https://make.wordpress.org/core/2020/03/03/wp-env-simple-local-environments-for-wordpress/) <br>

Reviewer 2: Since this is a post to announce that environment package from when it was introduced, I think it would be better to point to actual documentation on the package, e.g. `https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/`. However, that link is already present below under "Useful commands".

I think it would be better to omit this sentence, or we can simplify it to something like "See the section below for more details on how the environment works.".

> More details about getting started [here](./docs/Getting-started.md)

Reviewer: Add period to end of sentence

Author: 👍
